test_crates/inherent_associated_pub_const_missing/old/src/lib.rs

```diff
@@ -16,3 +18,20 @@ impl StructB {
     pub const PublicConstantRenameB: i32 = 0;
     const ConstantB: i32 = 0;
 }
+#[doc(hidden)]
+pub struct StructC {}
+impl StructC {
+    // should not be caught on removing since the struct #[doc(hidden)]
+    pub const PublicConstantF: i32 = 0;
+}
+//should not be caught by this lint on marking #[doc(hidden)]
+pub struct StructD {}
```

Comment:
Another formatting nitpick that isn't super important in this test code, but will be critical if working on the cargo-semver-checks source itself: it's good to stay consistent with style and formatting. `rustfmt` does a lot, but it can't do everything yet.

In this block I've highlighted, there are four mutually-inconsistent styles of comments: capitalized vs uncapitalized, each both seen with a space after the `//` and without a space. Ideally, everything should be `// Capitalized concise sentence.`.

Consistent style improves readability and gives readers a sense of ease and quality when looking at the code. If code looks messy, people will have negative impressions of it (and often, of its authors), even if the code itself works well.